airflow/configuration.py

```diff
@@ -458,6 +464,12 @@ def run_command(command):
 scheduler_heartbeat_sec = 5
 authenticate = true
 max_threads = 2
+
+# Turn off scheduler backfills by setting this to false.
+# Default behavior is unchanged
+# Command Line Backfills still work, but the scheduler
+# will not do backfills if this is false.
+backfill_by_default = true
```

Comment:
This section doesn't need to be here, the default value (as in `DEFAULT_CONFIG`) of `backfill_by_default` will be used if this entry

Even if we want to include it for clarity the comments should be dropped